packages/calcite-components/src/components/flow/flow.scss

```diff
@@ -1,61 +1,78 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden
+ *
+ * @prop --calcite-flow-animation-timing: Specifies the animation timing for the flow.
+ * @prop --calcite-flow-animation-opacity-start: Specifies the animation starting opacity for the flow.
+ * @prop --calcite-flow-animation-opacity-end: Specifies the animation ending opacity for the flow.
+ * @prop --calcite-flow-animation-advance: Specifies the advancing animation for the flow.
+ * @prop --calcite-flow-animation-retreat: Specifies the retreating animation for the flow.
+ */
+
 :host {
-  @extend %component-host;
-  @apply relative
-    flex
-    w-full
-    flex-auto
-    items-stretch
-    overflow-hidden
-    bg-transparent;
+  @include base-host();
+  position: relative;
+  display: flex;
+  inline-size: 100%;
+  flex: 1 1 auto;
+  align-items: stretch;
+  overflow: hidden;
+
+  --calcite-flow-animation-opacity-start: var(--calcite-opacity-50);
```

Comment:
Should these be "public" or `--calcite-internal` until we get requests to customize?

Reply:
Yeah I can make that change